tests/raftstore/pd.rs

```diff
@@ -193,7 +192,8 @@ impl Cluster {
         let conf_ver = region.get_region_epoch().get_conf_ver();
         let end_key = enc_end_key(&region);
 
-        let cur_region = self.get_region_by_id(region.get_id()).unwrap();
+        // it can pass handle_heartbeat_version means it must exist.
```

Comment:
Not a good style to assert the binding to another function `handle_heartbeat_version` here, in order to unwrap() directly.